packages/logic-utils/test/utils/xml-ns-key-spec.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
```

Comment:
its one thing to copy a one line regexp that should be elsewhere, but when we
also implement tests for that, does it become a larger tech debt?

Reply:
Wherever these functions are implemented they need tests...

Reply:
Yes the tests are needed, the point was that before we had a little duplicate regexp/logic now we have a full set of tests possibly in wrong place.